common/utils/searchResults.utils.ts

```diff
@@ -43,11 +65,27 @@ export const sortSearchByPosition = (
   search: string,
   results: SearchResult[],
 ): SearchResult[] => {
-  const res = sortSearchByTopic(results);
-  return res.toSorted((a, b) => {
+  // const res = sortSearchByTopic(results);
```

Comment:
Can this be removed or will we need it later?

Reply:
removed